Aspire Budgeting/Views/Dashboard/DashboardCardView.swift

```diff
@@ -1,39 +1,124 @@
 //
-//  DashboardCardVIew.swift
+//  DashboardCardView.swift
 //  Aspire Budgeting
 //
 
 import SwiftUI
 
 struct DashboardCardView: View {
-  struct Totals {
-    var availableTotal: AspireNumber
-    var budgetedTotal: AspireNumber
-    var spentTotals: AspireNumber
-  }
 
-  let categoryName: String
-  let totals: Totals
-  let categoryRows: [Category]
+  let cardViewItem: BaseCardView.CardViewItem
+  let colorInfo: BaseCardView.ColorInfo
+
+  private var gradient: LinearGradient {
+    Color.fondGradientFrom(startColor: colorInfo.gradientStartColor,
+                           endColor: colorInfo.gradientEndColor)
+  }
+  
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/6c344c52764dd09f4d37a8501914d1161cb941a9/Aspire Budgeting/Views/Dashboard/DashboardCardView.swift#L17"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>DashboardCardView.swift:17</code></td>
    </tr>
  </tbody>
</table>